sdk/keyvault/keyvault-secrets/samples/v4-beta/javascript/README.md

```diff
@@ -0,0 +1,75 @@
+---
+page_type: sample
```

Comment:
Wait, are we sure we want to add all of these files? normally these files are autogenerated from samples-dev. I could be missing something.

Reply:
For context we recently changed our doc generation scripts to generate beta documentation for beta packages. Since a beta is in progress here `dev-tool samples publish` publishes them under the v4-beta directory.

I spoke to @diberry offline and since we're interested in updating the GA docs I think the best and easiest thing to do would be:
1. Update samples-dev/helloWorld.ts
2. Manually update samples/v4/javascript/helloWorld.js and samples/v4/typescript/src/helloWorld.ts 

Commit these changes.